config/initializers/kaminari_config.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 Kaminari.configure do |config|
-  config.default_per_page = 5
+  config.default_per_page = 10
```

Comment:
Why is this change here?

Reply:
because filip asked for that change of pagination count a while back. This is that change that he asked for.